wait-for

```diff
@@ -1,7 +1,19 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
Won't this mean it's no longer:
> It is sh and alpine compatible.

?

Reply:
Now that you mention it, I'm not entirely sure.  I will add a docker test for it and see what happens

Reply:
I needed bash for a test, but I've switched to a POSIX-compliant one and can roll back this change.